e2e/sites.test.ts

```diff
@@ -25,7 +25,7 @@ test.describe( 'Servers', () => {
 		await onboarding.continueButton.click();
 
 		const whatsNewModal = new WhatsNewModal( session.mainWindow );
-		if ( await whatsNewModal.locator.isVisible( { timeout: 5000 } ) ) {
+		if ( await whatsNewModal.locator.isVisible( { timeout: 20000 } ) ) {
```

Comment:
This is causing some tests to be flaky.
It took me a while to find the cause for it since it never happened locally.

One solution I can think of is skipping the modal in the app if we are running e2e.
If anyone has any better suggestions please let me know.

Reply:
I ended up implementing a way around it for E2E in https://github.com/Automattic/studio/pull/1372/commits/703613b87a2d30e937d2c100f8f928451eb7d360
Let me know what you think.